pallets/accounts/src/lib.rs

```diff
@@ -159,10 +160,8 @@ pub struct EthLock<TBalance, TBlockNum> {
 	pub amount: TBalance,
 	/// Clamor Block Number in which the locked FRAG tokens was "effectively transfered" to the Clamor Blockchain
 	pub block_number: TBlockNum,
-	/// The block number when FRAG were locked the first time
-	pub first_lock_block_number: TBlockNum,
 	/// The FRAG lock period chosen by the user on Ethereum and received from the Lock event
-	pub lock_period: U256,
+	pub lock_period: U128,
```

Comment:
isn't this still too big? Maybe I'm wrong, but isn't this the enum?

Reply:
Yes it's the enum, but I have issues here with the encoding done by the `ethabi` library to regenerate the Ethereum message. Specifically this part was too difficult to make it right using types other than U256 and U128
```
ethabi::encode(
	&[
		Token::Bytes(lock.data.signature.0.to_vec()),
		Token::Uint(lock.data.amount),
		Token::Uint(U256::from(lock.data.lock_period))
	]
),
```

Reply:
Signature is fine, no need to store it u256 or 128 though.

Reply:
I changed it to `u8` (as it is sent from Ethereum).